capa/features/extractors/dnfile/insn.py

```diff
@@ -80,6 +83,15 @@ def extract_insn_api_features(fh: FunctionHandle, bh, ih: InsnHandle) -> Iterato
     if callee is None:
         return
 
+    if callee.methodname.startswith("get_") or callee.methodname.startswith("set_"):
+        if Token(ih.inner.operand.value).table == 6:
```

Comment:
Let's comment this check

Reply:
Added the comment in https://github.com/mandiant/capa/pull/1122/commits/ccb1aad09e64d53ef65108182d67bef00069c5e7